packages/slate-code/docs/slate-code.mdx

```diff
@@ -42,10 +45,26 @@ const plugins = [Code({
   }
 })]
 
+function handleKeyDown(event, editor, next) {
+  if (event.ctrlKey) {
+    switch (event.key) {
+      case 'y':
+        event.preventDefault();
+        editor.insertCode(editor);
```

Comment:
You shouldn't have to pass the `editor` here; Slate handles that automatically. It might be good to show passing some code in the example. Something like:
```
editor.insertCode({ code: "<h1>Hello</h1>" })
```